multisrc/overrides/madara/mangaproz/src/MangaPro.kt

```diff
@@ -0,0 +1,16 @@
+package eu.kanade.tachiyomi.extension.ar.mangaproz
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import okhttp3.OkHttpClient
+import java.util.concurrent.TimeUnit
+
+class MangaPro : Madara("Manga Pro", "https://manga-pro.com", "ar") {
+    override val id: Long = 964048798769065340
```

Comment:
Probably the URL structure have also changed and it's ideally better to bump the `versionId` to force users to migrate to update the URLs.

```suggestion
    // Theme changed from MangaThemesia to Madara.
    override val versionId = 2
```